src/screens/exposureHistory/RecentExposureView.tsx

```diff
@@ -26,10 +26,10 @@ export const RecentExposureScreen = () => {
   const i18n = useI18n();
 
   const navigation = useNavigation();
-  const back = useCallback(() => navigation.goBack(), [navigation]);
+  const close = useCallback(() => navigation.navigate('Home'), [navigation]);
   return (
     <SafeAreaView style={styles.flex}>
-      <Toolbar title="" navIcon="icon-back-arrow" navText={i18n.translate('PlacesLog.Back')} onIconClicked={back} />
+      <Toolbar2 navText={i18n.translate('DataUpload.Close')} showBackButton onIconClicked={close} />
```

Comment:
Let's rename this to make it more clear as to the difference.

Reply:
Good idea.